common/test/acceptance/tests/discussion/test_discussion_management.py

```diff
@@ -271,3 +288,163 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
 
         # category should not be selected.
         self.assertFalse(self.discussion_management_page.is_category_selected())
+
+
+@attr(shard=6)
+class DivisionSchemeTest(BaseDividedDiscussionTest, BaseDiscussionMixin):
+    """
+    Tests for changing the division scheme for Discussions.
+    """
+
+    def add_enrollment_modes(self, modes):
```

Comment:
Yeah, I thought about it. In this case and the other you mentioned, I just put in the comments the "extra" things that it does. Self-documenting method names are certainly preferable, but they would get quite long (and this is only a test helper method). I don't feel strongly either way.